### DISCUSSION TITLE
gate wallet not supported for version 1.0.x

### DISCUSSION THREAD

**Discussion by DAppArtisan** [rainbow-me/rainbowkit#1955] [General]
gate wallet not supported for version 1.0.x. how to added in old rainbowkit veriosn

**Comment by 0xmkh** [comment]
@DAppArtisan Gate wallet is only supported in 2.x version. Please upgrade to our latest 2.x version in order to add Gate wallet.

**Comment by DAppArtisan** [comment]
Thank you for your response.
